pom.xml

```diff
@@ -246,6 +227,15 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>java11-release8</id>
+            <activation>
+                <jdk>[11,</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>8</maven.compiler.release>
```

Comment:
isn't this going to be in the parent/root pom?

Reply:
Yes